addons/web/static/src/legacy/js/fields/field_utils.js

```diff
@@ -766,25 +768,25 @@ export default {
         json: formatJson,
     },
     parse: {
-        binary: _.identity,
-        boolean: _.identity, // todo
-        char: _.identity, // todo
+        binary: (value) => value,
+        boolean: (value) => value, // todo
```

Comment:
I would have factorized that in a small local identity function, this is only a style comment

Reply:
You don't like it like this ? :D